flake.nix

```
@@ -80,7 +102,7 @@
           playwright-driver.browsers
           postgresql_16
           protobuf
-          protoc-gen-go
+          customProtocGenGo
```

Let's follow the convention above and call it:

```suggestion
          proto_gen_go_1_30
```